src/routes/categories/components/Breadcrumbs.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Link } from 'redux-little-router'
+
+import CategoryModel from 'modules/endpoint/models/CategoryModel'
+import LocationModel from 'modules/endpoint/models/LocationModel'
+
+import style from './Breadcrumbs.css'
+
+/**
+ * Displays breadcrumbs (Links) for lower category levels
+ */
+class Breadcrumbs extends React.Component {
+  static propTypes = {
+    categories: PropTypes.arrayOf(PropTypes.instanceOf(CategoryModel)).isRequired,
+    category: PropTypes.instanceOf(CategoryModel).isRequired,
+    locations: PropTypes.arrayOf(PropTypes.instanceOf(LocationModel)).isRequired
+  }
+
+  /**
+   *
+   * @param location
+   */
+  getLocationTitle = (location) => (this.props.locations.find(_location => location === _location.code)).name
```

Comment:
It's possible that find() returns undefined to better catch that case and return null.

Reply:
done